src/libCps/Amount.h

```diff
@@ -49,6 +49,17 @@ class Amount final {
   constexpr auto operator>(const Amount& other) const {
     return !(*this <= other);
   }
+  constexpr auto operator+(const Amount& rhs) const {
+    // return the sum of two numbers as biggest type to avoid overflow
```

Comment:
Maybe I'm misunderstanding the comment - How are you avoiding overflow?

Reply:
I thought something like this: if you use toQa you downcast 256->128 so you could concevably overflow the sum. However, now that I look again, there is only a constructor for 256 (128 in variant unused?)... so it all seems moot.